### ISSUE TITLE
[ACCESSIBILITY] Unable to keyboard navigate to the Wellness Dropdown menu

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#2327]
Describe the accessibility issue
When gesturing through app with screen reader, once the user is on wellness, when they keep swiping, they completely skip the dropdown menu. On July 15th this issue did not exist. I'm unclear when it was introduced since then, but this prevents the proper retesting of bug 2023. 

To Reproduce
1. Turn on screen reader
2. toggle to Wellness
3. Swipe through this screen, the dropdown menu will be skipped and never be given focus. 

Screenshots
Video attached. 

https://user-images.githubusercontent.com/82892873/187456618-0ec78936-285e-45ed-8c65-619117536b64.mp4

Smartphone (please complete the following information):
Google Pixel 4a Android 12

Additional context

**Comment by TodorBachvarov** [comment]
Fixed in version 4.1.30.

**Comment by acovher2** [comment]
This is verified as sort of fixed in 4.1.30. When the user is gesturing into the dropdown and selects to expand the options, they are taken to the top lefthand I logo instead of starting on the top dropdown menu item. This causes them to need to gesture an extra 5 unnecessary times. I can open a separate bug for that or you can improve it with this bug. What do you prefer?

**Comment by TodorBachvarov** [comment]
@acovher2 I've tried but not able to fix the focus issue. This is known flutter issue, that we can't change the Accessibility focus programatically. So please close this issue as the original problem was solved.
